# Store Sale Data Analysis

## Data Preparation and Exploration

**Import store data**

In [1]:
import pandas as pd
df = pd.read_csv("store-data.csv")

**Check the top 10 rows**

In [2]:
df.head(10)

Row ID 
 Order ID 
 Order Date 
 Ship Date 
 Ship Mode 
 Customer ID 
 Customer Name 
 Segment 
 Country/Region 
 City 
 ... 
 Postal Code 
 Region 
 Product ID 
 Category 
 Sub-Category 
 Product Name 
 Sales 
 Quantity 
 Discount 
 Profit 
 
 
 
 
 0 
 1 
 CA-2019-152156 
 11/8/2019 
 11/11/2019 
 Second Class 
 CG-12520 
 Claire Gute 
 Consumer 
 United States 
 Henderson 
 ... 
 42420.0 
 South 
 FUR-BO-10001798 
 Furniture 
 Bookcases 
 Bush Somerset Collection Bookcase 
 261.9600 
 2 
 0.00 
 41.9136 
 
 
 1 
 2 
 CA-2019-152156 
 11/8/2019 
 11/11/2019 
 Second Class 
 CG-12520 
 Claire Gute 
 Consumer 
 United States 
 Henderson 
 ... 
 42420.0 
 South 
 FUR-CH-10000454 
 Furniture 
 Chairs 
 Hon Deluxe Fabric Upholstered Stacking Chairs,... 
 731.9400 
 3 
 0.00 
 219.5820 
 
 
 2 
 3 
 CA-2019-138688 
 6/12/2019 
 6/16/2019 
 Second Class 
 DV-13045 
 Darrin Van Huff 
 Corporate 
 United States 
 Los Angeles 
 ... 
 90036.0 
 West 
 OFF-LA-10000240 
 Office Supplies 
 Labels 
 Self-Adhesive Address Labels for Typewriters b... 
 14.6200 
 2 
 0.00 
 6.8714 
 
 
 3 
 4 
 US-2018-108966 
 10/11/2018 
 10/18/2018 
 Standard Class 
 SO-20335 
 Sean O'Donnell 
 Consumer 
 United States 
 Fort Lauderdale 
 ... 
 33311.0 
 South 
 FUR-TA-10000577 
 Furniture 
 Tables 
 Bretford CR4500 Series Slim Rectangular Table 
 957.5775 
 5 
 0.45 
 -383.0310 
 
 
 4 
 5 
 US-2018-108966 
 10/11/2018 
 10/18/2018 
 Standard Class 
 SO-20335 
 Sean O'Donnell 
 Consumer 
 United States 
 Fort Lauderdale 
 ... 
 33311.0 
 South 
 OFF-ST-10000760 
 Office Supplies 
 Storage 
 Eldon Fold 'N Roll Cart System 
 22.3680 
 2 
 0.20 
 2.5164 
 
 
 5 
 6 
 CA-2017-115812 
 6/9/2017 
 6/14/2017 
 Standard Class 
 BH-11710 
 Brosina Hoffman 
 Consumer 
 United States 
 Los Angeles 
 ... 
 90032.0 
 West 
 FUR-FU-10001487 
 Furniture 
 Furnishings 
 Eldon Expressions Wood and Plastic Desk Access... 
 48.8600 
 7 
 0.00 
 14.1694 
 
 
 6 
 7 
 CA-2017-115812 
 6/9/2017 
 6/14/2017 
 Standard Class 
 BH-11710 
 Brosina Hoffman 
 Consumer 
 United States 
 Los Angeles 
 ... 
 90032.0 
 West 
 OFF-AR-10002833 
 Office Supplies 
 Art 
 Newell 322 
 7.2800 
 4 
 0.00 
 1.9656 
 
 
 7 
 8 
 CA-2017-115812 
 6/9/2017 
 6/14/2017 
 Standard Class 
 BH-11710 
 Brosina Hoffman 
 Consumer 
 United States 
 Los Angeles 
 ... 
 90032.0 
 West 
 TEC-PH-10002275 
 Technology 
 Phones 
 Mitel 5320 IP Phone VoIP phone 
 907.1520 
 6 
 0.20 
 90.7152 
 
 
 8 
 9 
 CA-2017-115812 
 6/9/2017 
 6/14/2017 
 Standard Class 
 BH-11710 
 Brosina Hoffman 
 Consumer 
 United States 
 Los Angeles 
 ... 
 90032.0 
 West 
 OFF-BI-10003910 
 Office Supplies 
 Binders 
 DXL Angle-View Binders with Locking Rings by S... 
 18.5040 
 3 
 0.20 
 5.7825 
 
 
 9 
 10 
 CA-2017-115812 
 6/9/2017 
 6/14/2017 
 Standard Class 
 BH-11710 
 Brosina Hoffman 
 Consumer 
 United States 
 Los Angeles 
 ... 
 90032.0 
 West 
 OFF-AP-10002892 
 Office Supplies 
 Appliances 
 Belkin F5C206VTEL 6 Outlet Surge 
 114.9000 
 5 
 0.00 
 34.4700 
 
 
 
 10 rows × 21 columns

**Check data's shape**

In [3]:
df.shape

(9994, 21)

**Check data's information**

In [4]:
df.info()

<class 'pandas.core.frame.DataFrame'>
RangeIndex: 9994 entries, 0 to 9993
Data columns (total 21 columns):
 #   Column          Non-Null Count  Dtype  
---  ------          --------------  -----  
 0   Row ID          9994 non-null   int64  
 1   Order ID        9994 non-null   object 
 2   Order Date      9994 non-null   object 
 3   Ship Date       9994 non-null   object 
 4   Ship Mode       9994 non-null   object 
 5   Customer ID     9994 non-null   object 
 6   Customer Name   9994 non-null   object 
 7   Segment         9994 non-null   object 
 8   Country/Region  9994 non-null   object 
 9   City            9994 non-null   object 
 10  State           9994 non-null   object 
 11  Postal Code     9983 non-null   float64
 12  Region          9994 non-null   object 
 13  Product ID      9994 non-null   object 
 14  Category        9994 non-null   object 
 15  Sub-Category    9994 non-null   object 
 16  Product Name    9994 non-null   object 
 17  Sales           9994 non-null   f

-> 'Postal code' has missing value value 

**Count missing value in 'Postal code' column**

In [5]:
df['Postal Code'].isna().sum()

11

**Check the rows that have missing value**

In [6]:
df[ df['Postal Code'].isna() ]

Row ID 
 Order ID 
 Order Date 
 Ship Date 
 Ship Mode 
 Customer ID 
 Customer Name 
 Segment 
 Country/Region 
 City 
 ... 
 Postal Code 
 Region 
 Product ID 
 Category 
 Sub-Category 
 Product Name 
 Sales 
 Quantity 
 Discount 
 Profit 
 
 
 
 
 2234 
 2235 
 CA-2020-104066 
 12/5/2020 
 12/10/2020 
 Standard Class 
 QJ-19255 
 Quincy Jones 
 Corporate 
 United States 
 Burlington 
 ... 
 NaN 
 East 
 TEC-AC-10001013 
 Technology 
 Accessories 
 Logitech ClearChat Comfort/USB Headset H390 
 205.03 
 7 
 0.0 
 67.6599 
 
 
 5274 
 5275 
 CA-2018-162887 
 11/7/2018 
 11/9/2018 
 Second Class 
 SV-20785 
 Stewart Visinsky 
 Consumer 
 United States 
 Burlington 
 ... 
 NaN 
 East 
 FUR-CH-10000595 
 Furniture 
 Chairs 
 Safco Contoured Stacking Chairs 
 715.20 
 3 
 0.0 
 178.8000 
 
 
 8798 
 8799 
 US-2019-150140 
 4/6/2019 
 4/10/2019 
 Standard Class 
 VM-21685 
 Valerie Mitchum 
 Home Office 
 United States 
 Burlington 
 ... 
 NaN 
 East 
 TEC-PH-10002555 
 Technology 
 Phones 
 Nortel Meridian M5316 Digital phone 
 1294.75 
 5 
 0.0 
 336.6350 
 
 
 9146 
 9147 
 US-2019-165505 
 1/23/2019 
 1/27/2019 
 Standard Class 
 CB-12535 
 Claudia Bergmann 
 Corporate 
 United States 
 Burlington 
 ... 
 NaN 
 East 
 TEC-AC-10002926 
 Technology 
 Accessories 
 Logitech Wireless Marathon Mouse M705 
 99.98 
 2 
 0.0 
 42.9914 
 
 
 9147 
 9148 
 US-2019-165505 
 1/23/2019 
 1/27/2019 
 Standard Class 
 CB-12535 
 Claudia Bergmann 
 Corporate 
 United States 
 Burlington 
 ... 
 NaN 
 East 
 OFF-AR-10003477 
 Office Supplies 
 Art 
 4009 Highlighters 
 8.04 
 6 
 0.0 
 2.7336 
 
 
 9148 
 9149 
 US-2019-165505 
 1/23/2019 
 1/27/2019 
 Standard Class 
 CB-12535 
 Claudia Bergmann 
 Corporate 
 United States 
 Burlington 
 ... 
 NaN 
 East 
 OFF-ST-10001526 
 Office Supplies 
 Storage 
 Iceberg Mobile Mega Data/Printer Cart 
 1564.29 
 13 
 0.0 
 406.7154 
 
 
 9386 
 9387 
 US-2020-127292 
 1/19/2020 
 1/23/2020 
 Standard Class 
 RM-19375 
 Raymond Messe 
 Consumer 
 United States 
 Burlington 
 ... 
 NaN 
 East 
 OFF-PA-10000157 
 Office Supplies 
 Paper 
 Xerox 191 
 79.92 
 4 
 0.0 
 37.5624 
 
 
 9387 
 9388 
 US-2020-127292 
 1/19/2020 
 1/23/2020 
 Standard Class 
 RM-19375 
 Raymond Messe 
 Consumer 
 United States 
 Burlington 
 ... 
 NaN 
 East 
 OFF-PA-10001970 
 Office Supplies 
 Paper 
 Xerox 1881 
 12.28 
 1 
 0.0 
 5.7716 
 
 
 9388 
 9389 
 US-2020-127292 
 1/19/2020 
 1/23/2020 
 Standard Class 
 RM-19375 
 Raymond Messe 
 Consumer 
 United States 
 Burlington 
 ... 
 NaN 
 East 
 OFF-AP-10000828 
 Office Supplies 
 Appliances 
 Avanti 4.4 Cu. Ft. Refrigerator 
 542.94 
 3 
 0.0 
 152.0232 
 
 
 9389 
 9390 
 US-2020-127292 
 1/19/2020 
 1/23/2020 
 Standard Class 
 RM-19375 
 Raymond Messe 
 Consumer 
 United States 
 Burlington 
 ... 
 NaN 
 East 
 OFF-EN-10001509 
 Office Supplies 
 Envelopes 
 Poly String Tie Envelopes 
 2.04 
 1 
 0.0 
 0.9588 
 
 
 9741 
 9742 
 CA-2018-117086 
 11/8/2018 
 11/12/2018 
 Standard Class 
 QJ-19255 
 Quincy Jones 
 Corporate 
 United States 
 Burlington 
 ... 
 NaN 
 East 
 FUR-BO-10004834 
 Furniture 
 Bookcases 
 Riverside Palais Royal Lawyers Bookcase, Royal... 
 4404.90 
 5 
 0.0 
 1013.1270 
 
 
 
 11 rows × 21 columns

**Fill missing value with actual postcode of Burlington, Vermont, USA (05401)**

In [7]:
df['Postal Code'] = df['Postal Code'].fillna("05401")

**Check data's information again**

In [8]:
df.info()
df['Postal Code'].isna().sum()

<class 'pandas.core.frame.DataFrame'>
RangeIndex: 9994 entries, 0 to 9993
Data columns (total 21 columns):
 #   Column          Non-Null Count  Dtype  
---  ------          --------------  -----  
 0   Row ID          9994 non-null   int64  
 1   Order ID        9994 non-null   object 
 2   Order Date      9994 non-null   object 
 3   Ship Date       9994 non-null   object 
 4   Ship Mode       9994 non-null   object 
 5   Customer ID     9994 non-null   object 
 6   Customer Name   9994 non-null   object 
 7   Segment         9994 non-null   object 
 8   Country/Region  9994 non-null   object 
 9   City            9994 non-null   object 
 10  State           9994 non-null   object 
 11  Postal Code     9994 non-null   object 
 12  Region          9994 non-null   object 
 13  Product ID      9994 non-null   object 
 14  Category        9994 non-null   object 
 15  Sub-Category    9994 non-null   object 
 16  Product Name    9994 non-null   object 
 17  Sales           9994 non-null   f

0

**Export csv data for visualization in Tableau dashboard**

In [11]:
df.to_csv('store-data-cleaned.csv')

Explore more data in statistics detail

In [9]:
df.describe(include = "all")

Row ID 
 Order ID 
 Order Date 
 Ship Date 
 Ship Mode 
 Customer ID 
 Customer Name 
 Segment 
 Country/Region 
 City 
 ... 
 Postal Code 
 Region 
 Product ID 
 Category 
 Sub-Category 
 Product Name 
 Sales 
 Quantity 
 Discount 
 Profit 
 
 
 
 
 count 
 9994.000000 
 9994 
 9994 
 9994 
 9994 
 9994 
 9994 
 9994 
 9994 
 9994 
 ... 
 9994.0 
 9994 
 9994 
 9994 
 9994 
 9994 
 9994.000000 
 9994.000000 
 9994.000000 
 9994.000000 
 
 
 unique 
 NaN 
 5009 
 1236 
 1334 
 4 
 793 
 793 
 3 
 1 
 531 
 ... 
 631.0 
 4 
 1862 
 3 
 17 
 1849 
 NaN 
 NaN 
 NaN 
 NaN 
 
 
 top 
 NaN 
 CA-2020-100111 
 9/5/2019 
 12/16/2018 
 Standard Class 
 WB-21850 
 William Brown 
 Consumer 
 United States 
 New York City 
 ... 
 10035.0 
 West 
 OFF-PA-10001970 
 Office Supplies 
 Binders 
 Staple envelope 
 NaN 
 NaN 
 NaN 
 NaN 
 
 
 freq 
 NaN 
 14 
 38 
 35 
 5968 
 37 
 37 
 5191 
 9994 
 915 
 ... 
 263.0 
 3203 
 19 
 6026 
 1523 
 48 
 NaN 
 NaN 
 NaN 
 NaN 
 
 
 mean 
 4997.500000 
 NaN 
 NaN 
 NaN 
 NaN 
 NaN 
 NaN 
 NaN 
 NaN 
 NaN 
 ... 
 NaN 
 NaN 
 NaN 
 NaN 
 NaN 
 NaN 
 229.858001 
 3.789574 
 0.156203 
 28.656896 
 
 
 std 
 2885.163629 
 NaN 
 NaN 
 NaN 
 NaN 
 NaN 
 NaN 
 NaN 
 NaN 
 NaN 
 ... 
 NaN 
 NaN 
 NaN 
 NaN 
 NaN 
 NaN 
 623.245101 
 2.225110 
 0.206452 
 234.260108 
 
 
 min 
 1.000000 
 NaN 
 NaN 
 NaN 
 NaN 
 NaN 
 NaN 
 NaN 
 NaN 
 NaN 
 ... 
 NaN 
 NaN 
 NaN 
 NaN 
 NaN 
 NaN 
 0.444000 
 1.000000 
 0.000000 
 -6599.978000 
 
 
 25% 
 2499.250000 
 NaN 
 NaN 
 NaN 
 NaN 
 NaN 
 NaN 
 NaN 
 NaN 
 NaN 
 ... 
 NaN 
 NaN 
 NaN 
 NaN 
 NaN 
 NaN 
 17.280000 
 2.000000 
 0.000000 
 1.728750 
 
 
 50% 
 4997.500000 
 NaN 
 NaN 
 NaN 
 NaN 
 NaN 
 NaN 
 NaN 
 NaN 
 NaN 
 ... 
 NaN 
 NaN 
 NaN 
 NaN 
 NaN 
 NaN 
 54.490000 
 3.000000 
 0.200000 
 8.666500 
 
 
 75% 
 7495.750000 
 NaN 
 NaN 
 NaN 
 NaN 
 NaN 
 NaN 
 NaN 
 NaN 
 NaN 
 ... 
 NaN 
 NaN 
 NaN 
 NaN 
 NaN 
 NaN 
 209.940000 
 5.000000 
 0.200000 
 29.364000 
 
 
 max 
 9994.000000 
 NaN 
 NaN 
 NaN 
 NaN 
 NaN 
 NaN 
 NaN 
 NaN 
 NaN 
 ... 
 NaN 
 NaN 
 NaN 
 NaN 
 NaN 
 NaN 
 22638.480000 
 14.000000 
 0.800000 
 8399.976000 
 
 
 
 11 rows × 21 columns

After exploring the data set, there are **4 delivery modes**, **3 customer segments**, and **3 item categories** with **17 subcategories**. In addition, orders come from **4 regions** of the United States, **49 states**, and **531 cities**.


In addition, I have **5 business question** regarding the data set.

- Which categories are the most profitable and best-selling?
- Which subcategories are the most profitable and best-selling?
- In which city is the sales volume greatest?
- Which segment of customers generates the most profit?
- Who are the top twenty profit-generating customers?

## Data Analysis

**Which categories are the most profitable and best-selling?**

In [10]:
df_category = pd.DataFrame(df.groupby(['Category'])[['Quantity', 'Sales', 'Profit']].sum().reset_index())
df_category

Category 
 Quantity 
 Sales 
 Profit 
 
 
 
 
 0 
 Furniture 
 8028 
 741999.7953 
 18451.2728 
 
 
 1 
 Office Supplies 
 22906 
 719047.0320 
 122490.8008 
 
 
 2 
 Technology 
 6939 
 836154.0330 
 145454.9481

**Which sub-categories are the most profitable and best-selling?**

In [11]:
df_subcat = pd.DataFrame(df.groupby(['Sub-Category'])[['Quantity', 'Sales', 'Profit']].sum())

# Most Quantity ofsales
df_subcat.sort_values('Quantity', ascending = False).reset_index()

Sub-Category 
 Quantity 
 Sales 
 Profit 
 
 
 
 
 0 
 Binders 
 5974 
 203412.7330 
 30221.7633 
 
 
 1 
 Paper 
 5178 
 78479.2060 
 34053.5693 
 
 
 2 
 Furnishings 
 3563 
 91705.1640 
 13059.1436 
 
 
 3 
 Phones 
 3289 
 330007.0540 
 44515.7306 
 
 
 4 
 Storage 
 3158 
 223843.6080 
 21278.8264 
 
 
 5 
 Art 
 3000 
 27118.7920 
 6527.7870 
 
 
 6 
 Accessories 
 2976 
 167380.3180 
 41936.6357 
 
 
 7 
 Chairs 
 2356 
 328449.1030 
 26590.1663 
 
 
 8 
 Appliances 
 1729 
 107532.1610 
 18138.0054 
 
 
 9 
 Labels 
 1400 
 12486.3120 
 5546.2540 
 
 
 10 
 Tables 
 1241 
 206965.5320 
 -17725.4811 
 
 
 11 
 Fasteners 
 914 
 3024.2800 
 949.5182 
 
 
 12 
 Envelopes 
 906 
 16476.4020 
 6964.1767 
 
 
 13 
 Bookcases 
 868 
 114879.9963 
 -3472.5560 
 
 
 14 
 Supplies 
 647 
 46673.5380 
 -1189.0995 
 
 
 15 
 Machines 
 440 
 189238.6310 
 3384.7569 
 
 
 16 
 Copiers 
 234 
 149528.0300 
 55617.8249

In [38]:
# Best Selling
df_subcat.sort_values('Sales', ascending = False).reset_index()

Sub-Category 
 Quantity 
 Sales 
 Profit 
 
 
 
 
 0 
 Phones 
 3289 
 330007.0540 
 44515.7306 
 
 
 1 
 Chairs 
 2356 
 328449.1030 
 26590.1663 
 
 
 2 
 Storage 
 3158 
 223843.6080 
 21278.8264 
 
 
 3 
 Tables 
 1241 
 206965.5320 
 -17725.4811 
 
 
 4 
 Binders 
 5974 
 203412.7330 
 30221.7633 
 
 
 5 
 Machines 
 440 
 189238.6310 
 3384.7569 
 
 
 6 
 Accessories 
 2976 
 167380.3180 
 41936.6357 
 
 
 7 
 Copiers 
 234 
 149528.0300 
 55617.8249 
 
 
 8 
 Bookcases 
 868 
 114879.9963 
 -3472.5560 
 
 
 9 
 Appliances 
 1729 
 107532.1610 
 18138.0054 
 
 
 10 
 Furnishings 
 3563 
 91705.1640 
 13059.1436 
 
 
 11 
 Paper 
 5178 
 78479.2060 
 34053.5693 
 
 
 12 
 Supplies 
 647 
 46673.5380 
 -1189.0995 
 
 
 13 
 Art 
 3000 
 27118.7920 
 6527.7870 
 
 
 14 
 Envelopes 
 906 
 16476.4020 
 6964.1767 
 
 
 15 
 Labels 
 1400 
 12486.3120 
 5546.2540 
 
 
 16 
 Fasteners 
 914 
 3024.2800 
 949.5182

In [16]:
# Most Profitable
df_subcat.sort_values('Profit', ascending = False).reset_index()

Sub-Category 
 Quantity 
 Sales 
 Profit 
 
 
 
 
 0 
 Copiers 
 234 
 149528.0300 
 55617.8249 
 
 
 1 
 Phones 
 3289 
 330007.0540 
 44515.7306 
 
 
 2 
 Accessories 
 2976 
 167380.3180 
 41936.6357 
 
 
 3 
 Paper 
 5178 
 78479.2060 
 34053.5693 
 
 
 4 
 Binders 
 5974 
 203412.7330 
 30221.7633 
 
 
 5 
 Chairs 
 2356 
 328449.1030 
 26590.1663 
 
 
 6 
 Storage 
 3158 
 223843.6080 
 21278.8264 
 
 
 7 
 Appliances 
 1729 
 107532.1610 
 18138.0054 
 
 
 8 
 Furnishings 
 3563 
 91705.1640 
 13059.1436 
 
 
 9 
 Envelopes 
 906 
 16476.4020 
 6964.1767 
 
 
 10 
 Art 
 3000 
 27118.7920 
 6527.7870 
 
 
 11 
 Labels 
 1400 
 12486.3120 
 5546.2540 
 
 
 12 
 Machines 
 440 
 189238.6310 
 3384.7569 
 
 
 13 
 Fasteners 
 914 
 3024.2800 
 949.5182 
 
 
 14 
 Supplies 
 647 
 46673.5380 
 -1189.0995 
 
 
 15 
 Bookcases 
 868 
 114879.9963 
 -3472.5560 
 
 
 16 
 Tables 
 1241 
 206965.5320 
 -17725.4811

 **Which segment of customers generates the most profit?**

In [17]:
df_segment = pd.DataFrame(df.groupby(['Segment'])[['Profit']].sum().reset_index())
df_segment

Segment 
 Profit 
 
 
 
 
 0 
 Consumer 
 134119.2092 
 
 
 1 
 Corporate 
 91979.1340 
 
 
 2 
 Home Office 
 60298.6785

**Which cities are the top 10 sales volume greatest?**

In [18]:
df_city = pd.DataFrame(df.groupby(['City'])[['Sales', 'Quantity']].sum().sort_values('Sales',ascending = False).reset_index())
df_city.head(10)

City 
 Sales 
 Quantity 
 
 
 
 
 0 
 New York City 
 256368.1610 
 3417 
 
 
 1 
 Los Angeles 
 175851.3410 
 2879 
 
 
 2 
 Seattle 
 119540.7420 
 1590 
 
 
 3 
 San Francisco 
 112669.0920 
 1935 
 
 
 4 
 Philadelphia 
 109077.0130 
 1981 
 
 
 5 
 Houston 
 64504.7604 
 1466 
 
 
 6 
 Chicago 
 48539.5410 
 1132 
 
 
 7 
 San Diego 
 47521.0290 
 670 
 
 
 8 
 Jacksonville 
 44713.1830 
 429 
 
 
 9 
 Springfield 
 43054.3420 
 649

**Who are the top 20 profit-generating customers?**

In [19]:
# Calculate the profit grouped by Customer Name
df_customer = pd.DataFrame(df.groupby('Customer Name')['Profit'].sum())

# Order top 10 customers who make the most profit
df_customer = pd.DataFrame(df_customer.sort_values('Profit', ascending = False).round(2).reset_index().head(20))
df_customer

Customer Name 
 Profit 
 
 
 
 
 0 
 Tamara Chand 
 8981.32 
 
 
 1 
 Raymond Buch 
 6976.10 
 
 
 2 
 Sanjit Chand 
 5757.41 
 
 
 3 
 Hunter Lopez 
 5622.43 
 
 
 4 
 Adrian Barton 
 5444.81 
 
 
 5 
 Tom Ashbrook 
 4703.79 
 
 
 6 
 Christopher Martinez 
 3899.89 
 
 
 7 
 Keith Dawkins 
 3038.63 
 
 
 8 
 Andy Reiter 
 2884.62 
 
 
 9 
 Daniel Raglin 
 2869.08 
 
 
 10 
 Tom Boeckenhauer 
 2798.37 
 
 
 11 
 Nathan Mautz 
 2751.68 
 
 
 12 
 Sanjit Engle 
 2650.68 
 
 
 13 
 Bill Shonely 
 2616.06 
 
 
 14 
 Harry Marie 
 2437.98 
 
 
 15 
 Todd Sumrall 
 2371.71 
 
 
 16 
 Brian Moss 
 2199.28 
 
 
 17 
 Christopher Conant 
 2177.05 
 
 
 18 
 Jane Waco 
 2173.71 
 
 
 19 
 Helen Wasserman 
 2164.16